etc/WMAgentConfig.py

```diff
@@ -242,7 +242,7 @@
 config.ErrorHandler.maxRetries = maxJobRetries
 config.ErrorHandler.pollInterval = 240
 config.ErrorHandler.readFWJR = True
-config.ErrorHandler.failureExitCodes = [50660, 50661, 50664, 71102]
+config.ErrorHandler.failureExitCodes = [87, 8026, 50660, 50661, 50664, 71102]
```

Comment:
I couldn't find 87 below.
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/WMExceptions.py#L82
Should we need to add that?